middlewares/jimp.js

```diff
@@ -0,0 +1,18 @@
+const Jimp = require('jimp');
+const path = require('path');
+const uploadDir = path.join(process.cwd(), 'tmp');
+
+const resizeImage = async (req, res, next) => {
+  const { originalname } = req.file;
+  const imagePath = path.join(uploadDir, originalname);
+  const image = await Jimp.read(imagePath);
+  try {
+    await image.resize(250, 250).write(path.join(uploadDir, originalname));
+    console.log('Image resized and saved successfully.');
```

Comment:
Już wolał bym logowanie errora niż pozytywnego zapisu. Powiedzmy że masz jakiś turbo duży serwis to wtedy pojawi Ci się spory spam w logach, który wsumie nic nie daje....